scripts/tools/nrfconnect/tests/test_generate_factory_data.py

```diff
@@ -154,8 +154,9 @@ def test_generate_factory_data_all_specified(self):
                                    '--product_id', '0xABCD',
                                    '--vendor_name', 'Nordic Semiconductor ASA',
                                    '--product_name', 'Lock Gen2',
-                                   '--product_label', 'Lock',
                                    '--part_number', 'PCA10056',
+                                   '--product_url', 'https://nonexistentlockfactory.com/lock',
```

Comment:
nit: IANA reserved the domain to be used in documentation/examples: https://www.iana.org/domains/reserved

Maybe we can just use it: `https://example.com/lock`